canonical/cloud-init
# DO NOT MERGE: Ubuntu/jammy

# Upload point release 22.3.3 to Ubuntu Jammy. Do Not merge

## Additional context
This PR establishes a peer review process for Ubuntu downstream releases providing a place where visual diffs are easier to publicly review and comments and questions can be preserved prior to "acceptance" of these changes.

# Upon approval of this PR:
 1. an upstream tag `ubuntu/22.3.3-0ubuntu1_22.04.1 will be preserved to represent what was uploaded to Ubuntu for 22.04 (Jammy).
 2. This PR will be closed

# this branch was created using this process
1. git checkout upstream/ubuntu/jammy -B ubuntu/jammy; new-upstream-snapshot 22.3.3;
 2. Manually redact useless d/changelog lines about 22.3.2 and 22.3.1 releases as the end result of 22.3.3 is captured
 3. Manually change `New upstream snapshot` text to `New upstream bug fix release.`
 4. Update SRU bug to match our current process bug that is in progress

## Test Steps
build-package
sbuild package
check d/changelog content and version for proper reflection of git commits included in this point release
ability to quilt push -a; tox -e py3; quilt pop -a;
